benchmarks/serverRouteDispatch.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
```

Comment:
the work in this file should help to make it easier to test routers in general through Brisket.Testing